src/mosquitto/mod.rs

```diff
@@ -0,0 +1,59 @@
+use testcontainers::{core::WaitFor, Image, ImageArgs};
+
+/// Module to work with [`Mosquitto`] inside of tests.
+///
+/// Starts a MQTT broker without authentication.
+///
+///
+/// # Example
+/// ```
+/// use testcontainers::clients;
+/// use testcontainers_modules::mosquitto;
+///
+/// let docker = clients::Cli::default();
+/// let mosquitto_instance = docker.run(mosquitto::Mosquitto);
+///
+/// let broker_url = format!("127.0.0.1:{}", mosquitto_instance.get_host_port_ipv4(1883));
+/// ```
+///
+/// [`Mosquitto`]: https://mosquitto.org/
+/// [`Mosquitto docker image`]: https://hub.docker.com/_/eclipse-mosquitto
+
+const NAME: &str = "eclipse-mosquitto";
+const TAG: &str = "2.0.18";
+
+#[derive(Debug, Default, Clone)]
+pub struct Mosquitto;
+#[derive(Debug, Default, Clone)]
+pub struct MosquittoArgs;
+
+impl ImageArgs for MosquittoArgs {
+    fn into_iterator(self) -> Box<dyn Iterator<Item = String>> {
+        Box::new(
+            vec![
+                "mosquitto".to_string(),
+                "-c".to_string(),
+                "/mosquitto-no-auth.conf".to_string(),
```

Comment:
Can this be configured by environment variable instead of CLI arg? It would allow users to override the behavior with `RunnableImage`

Reply:
There is no config with env var apparently: https://github.com/eclipse/mosquitto/tree/master/docker/2.0#running-without-a-configuration-file